infrastructure/lambda/template.yaml

```diff
@@ -543,12 +559,51 @@ Resources:
     Condition: IsNotProdLikeEnvironment
     Properties:
       BatchSize: 10
-      Enabled: true
+      Enabled: False
```

Comment:
I think this is fine, I have a feeling we can remove this function entirely.

Reply:
So shall I take this enabled: False out of the PR?